src/main/java/com/salesforce/dynamodbv2/mt/mappers/sharedtable/impl/CachingAmazonDynamoDbStreams.java

```diff
@@ -87,36 +87,39 @@
         private long getRecordsLimitExceededBackoffInMillis =
                 DEFAULT_GET_RECORDS_LIMIT_EXCEEDED_BACKOFF_IN_MILLIS;
 
-        Builder(AmazonDynamoDBStreams amazonDynamoDbStreams) {
+        public Builder(AmazonDynamoDBStreams amazonDynamoDbStreams) {
```

Comment:
This triggers annoying "access can be package-private" warnings in Intellij.  I generally like having that warning enabled but in this case we don't want it.

Reply:
I had to make it public so that MtAmazonDynamoDbStreams (which is in another package) is able to access it, so hopefully we won't get warnings here. But I do see the warnings for the with* methods, since it doesn't use those.

Reply:
Yeah, this was more of a complaint about Intellij than a request to make a change.  :-)